experimental/saitypesextensions.h

```diff
@@ -104,5 +104,24 @@ typedef enum _sai_dash_tunnel_dscp_mode_t
 
 } sai_dash_tunnel_dscp_mode_t;
 
+/**
+ * @brief Defines a list of enums for dash_routing_actions
+ *
+ * @flags strict
+ */
+typedef enum _sai_dash_routing_actions_t
+{
+    SAI_DASH_ROUTING_ACTIONS_STATIC_ENCAP = 1,
+
+    SAI_DASH_ROUTING_ACTIONS_NAT = 2,
+
+    SAI_DASH_ROUTING_ACTIONS_NAT46 = 4,
+
+    SAI_DASH_ROUTING_ACTIONS_NAT64 = 8,
+
+    SAI_DASH_ROUTING_ACTIONS_NAT_PORT = 16,
+
+} sai_dash_routing_actions_t;
```

Comment:
can you use 1 << x as in other strict flags ?

Reply:
thanks Kamil and will do! i will fix the SAI generator in DASH, so we won't have this problem starting the very next type!